libs/output-mapping/src/Writer/Helper/DeleteWhereHelper.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Writer\Helper;
+
+class DeleteWhereHelper
```

Comment:
Škoda že se tu pracuje s raw configurací co ještě neprošla žádnou validací. Mohlo by to být jednodušší